components_library/boards/lupv_board.py

```diff
@@ -136,9 +155,8 @@ def __init__(
 
         self.lupio_pic.num_threads = self.processor.get_num_cores()
 
-        self.lupio_tmr.num_threads = self.processor.get_num_cores()
-
-        self.clint.num_threads = self.processor.get_num_cores()
+#        self.clint.num_threads = self.processor.get_num_cores()
+        self.clint.num_threads = 0
```

Comment:
That's probably incorrect but the fact that it somewhat works may be caused by the bugs in connectivity mentioned above.